nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -179,62 +169,95 @@ class WeightNodeParams:
     compression_config = WeightCompressionConfig()
 
 
-def _int8_compress(
-    weight: np.ndarray, reduction_axes: Union[int, Tuple[int]]
+def _do_integer_quantization(
```

Comment:
The name of the function does not reflect the fact that it supports NF4 as well.